packages/crml-cennzx-spot/src/derives/assetWithdrawn.ts

```diff
@@ -13,48 +13,50 @@
 // limitations under the License.
 
 import {ApiInterfaceRx} from '@cennznet/api/types';
-import {AnyAssetId, AnyNumber} from '@cennznet/types/types';
+import {AnyNumber} from '@cennznet/types/types';
+import {AssetId} from '@cennznet/types';
```

Comment:
Could we understand the differences between '@cennznet/types/types' and '@cennznet/types', any chance to distinguish those two packages by names or even combined into one?